server/services/offender/responses/approvedVisitors.js

```diff
@@ -0,0 +1,17 @@
+const { capitalizePersonName } = require('../../../utils/index');
+
+const sortContacts = ({ createDateTime: a }, { createDateTime: b }) => {
+  if (a < b) return -1;
+  return a > b ? 1 : 0;
```

Comment:
There's a few edge cases around lexigraphically sorting string timestamps..
I think this maybe ok though